### DISCUSSION TITLE
how do I create a new daily note in Zed

### DISCUSSION THREAD

**Discussion by Saltiola7** [Feel-ix-343/markdown-oxide#174] [General]
how do I create a new daily note in Zed?
there is no items in the command menu list for this, Im not sure if I got the installation right. there is no documentation for zed.

**Comment by Feel-ix-343** [comment]
Hello!

Since Zed doesn't support workspace commands, you cannot open daily notes!

I have an issue open for this on their repo; once that gets resolved this can too
